datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/DatasetComparison.scala

```diff
@@ -18,21 +18,22 @@ package za.co.absa.hermes.datasetComparison
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{StructField, StructType}
 import org.apache.spark.sql.{Column, DataFrame, SparkSession}
-import za.co.absa.hermes.datasetComparison.cliUtils.CliOptions
-import za.co.absa.hermes.datasetComparison.config.DatasetComparisonConfig
+import za.co.absa.commons.spark.SchemaUtils
+import za.co.absa.hermes.datasetComparison.config.{DatasetComparisonConfig, TypesafeConfig}
 import za.co.absa.hermes.utils.HelperFunctions
 
 /**
  * Class that is the brain of the DatasetComparison module. This class should be used in case of using DatasetComparison
  * as a library. In case of running the DatasetComparison as SparkJob, please use the DatasetComparisonJob.
  *
- * @param cliOptions Config object holding run based configurable parameters.
  * @param config Config object holding project based configurable parameters. Difference to the cliOptions is that these
```

Comment:
This class seems important enough to have its fields document individually.

Reply:
Should be fixed